### PR TITLE
Changelog Updates

### PR DESCRIPTION
**Pull request by sfsholden**
### What does this PR do?
Changelog entries for 47.2.0 release.

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+
+- Fix doc sidebar ([PR #1705](https://github.com/forcedotcom/salesforcedx-vscode/pull/1705))
+
+- Reorganize user guide section ([PR #1692](https://github.com/forcedotcom/salesforcedx-vscode/pull/1692))
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Expose the template input in project create for users working with Analytics settings ([PR #1712](https://github.com/forcedotcom/salesforcedx-vscode/pull/1712))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md]: I'm not sure if this was something we wanted to let people know about. It seemed like this was specific to WAVE and could be internal only.

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md]: We should keep it, it's not internal only since the changes to the project command will go out as part of the new cli build.

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- New Analytics project when running `SFDX: Create Project` ([PR #1712](https://github.com/forcedotcom/salesforcedx-vscode/pull/1712))
```
Maybe reword it so that it's easier to understand from a user perspective.

**Comment by sfsholden** [comment]
I am closing this PR and opening a new one. Something goofy happened with the rebase.
